test/e2e_node/oomkiller_linux_test.go

```diff
@@ -59,13 +80,27 @@ var _ = SIGDescribe("OOMKiller [LinuxOnly] [NodeConformance]", func() {
 		})
 	}
 	for _, tc := range testCases {
-		runOomKillerTest(f, tc)
+		runOomKillerTest(f, tc, "")
```

Comment:
It passes an empty string, so the kubeReservedMemory is not updated. Do you intend to add more cases later ?

Reply:
yes, I plan to add more test cases.